### ISSUE TITLE
Clicking 'Not Now' in FireFox for screen sharing requires refresh of client to subsequently share screen

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#11484]
**Describe the bug**
Clicking 'Not Now' in FireFox for screen sharing requires refresh of client to subsequently share screen

**To Reproduce**
1. Go to FireFox 86
2. Join https://test.bigbluebutton.org/
3. Click 'Share Screen'
4. Click 'Not Now'

**Expected behavior**
Screen is not shared, clicking Share Screen button would work.

**Actual behavior**
An error occurs and user is unable to click Share Screen again without refreshing the browser.

**Screenshots**
![image](https://user-images.githubusercontent.com/290351/109008995-5c3a0900-7684-11eb-82f0-bbef793dd532.png)

**BBB version (optional):**
BigBlueButton 2.2
FireFox 86

**Comment by prlanzarin** [comment]
This is not an issue. It's by FF's design (or will).
The permission denial is **memorized** and it's indicated in the URL bar. See screenshot, right icon.
It also doesn't require a refresh. You can just click the URL bar icon and forget the memorized permission and try sharing the screen again.

![image](https://user-images.githubusercontent.com/4529051/109009680-93f97e80-768d-11eb-8d05-8dcbe3f0d1d8.png)

**Comment by basisbit** [comment]
I'd suggest looking at how Jitsi "solved" this: when the error happens, it shows a short tutorial of how the user can unblock screen recording, and it shows the correct instructions for the used webbrowser.
